WcaOnRails/app/models/competition.rb

```diff
@@ -379,6 +379,10 @@ def registration_full?
     competitor_limit_enabled? && registrations.accepted_and_paid_pending_count >= competitor_limit
   end
 
+  def number_of_bookmarks
+    bookmarked_users.length
```

Comment:
Nitpick: This should be `bookmarked_users.count` instead of `.length`. The `bookmarked_users` field is a Rails ORM association mapping (more or less) directly to SQL, but `length` is a general **Ruby** method. Doing it this way loads all users via `SELECT *` into a Ruby array and then counts the length of the array. Using `.count` does a SQL `SELECT COUNT(*)` directly.

Reply:
done!